src/test/java/orangetest/sqltask/ConnectionToDatabase.java

```diff
@@ -0,0 +1,11 @@
+package orangetest.sqltask;
+
+import java.sql.*;
+
+public class ConnectionToDatabase extends DataProvider {
```

Comment:
extends DataProvider makes no sense here

Reply:
fixed, moved fields to ConnectionToDatabase class